editor/js/Editor.js

```diff
@@ -659,7 +659,12 @@ Editor.prototype = {
 		var loader = new THREE.ObjectLoader();
 		var camera = await loader.parseAsync( json.camera );
 
+		const existingUuid = this.camera.uuid;
+		const incomingUuid = camera.uuid;
 		this.camera.copy( camera );
+		this.camera.uuid = incomingUuid;
+		delete this.cameras[ existingUuid ];
+		this.cameras[ incomingUuid ] = this.camera;
```

Comment:
This is somewhat hard-to-follow code. At least a comment should be added that explains why UUIDs are copied around.

Reply:
ok